config/metadata.yaml

```diff
@@ -0,0 +1,339 @@
+      package_name\n\n
+      The fields after package name override the corresponding data in the\n
+      first source of the package YAML.
+    default: []
+  packages_dir:
+    option_strings:
+    - --packages-dir
+    - -d
+    help: Directory of packages used in the ISO image
+    default: result/packages/latest
+  packages_metadata_repo_branch:
+    option_strings:
+    - --packages-metadata-repo-branch
+    help: Packages metadata git repository branch
+    default: master
+  packages_metadata_repo_refspecs:
+    option_strings:
+    - --packages-metadata-repo-refspecs
+    help: Packages metadata git repository refspecs to fetch. Refer
+      to https://git-scm.com/book/id/v2/Git-Internals-The-Refspec
+    default:
+    - +refs/heads/*:refs/remotes/origin/*
+    - +refs/pull/*:refs/remotes/origin/pr/*
+  packages_metadata_repo_url:
+    option_strings:
+    - --packages-metadata-repo-url
+    help: Packages metadata git repository URL
+    default: https://github.com/open-power-host-os/versions.git
+  pungi_args:
+    option_strings:
+    - --pungi-args
+    help: Arguments passed to pungi command
+    default: --isfinal --nosource --nodebuginfo
+  pungi_binary:
+    option_strings:
+    - --pungi-binary
+    help: Pungi binary path
+    default: /usr/bin/pungi
+  push_repo_branch:
+    option_strings:
+    - --push-repo-branch
+    help: Branch of the repository used for pushing
+    default: master
+  push_repo_url:
+    option_strings:
+    - --push-repo-url
+    help: URL of the repository used for pushing
+    default: ''
+  push_updates:
+    option_strings:
+    - --no-push-updates
+    help: Push file updates to remote repository
+    default: True
+    action: store_false
+  release_notes_repo_branch:
+    option_strings:
+    - --release-notes-repo-branch
+    help: Release notes repository branch
+    default: master
+  release_notes_repo_url:
+    option_strings:
+    - --release-notes-repo-url
+    help: Release notes repository URL
+    default: https://github.com/open-power-host-os/open-power-host-os.github.io.git
+  result_dir:
+    option_strings:
+    - --result-dir
+    - -r
+    help: Directory used to store results
+    default: result
+  update_packages_repos_before_build:
+    option_strings:
+    - --no-update-packages-repos-before-build
+    help: Do not update code repositories before building
+    default: True
+    action: store_false
+  updater_email:
+    option_strings:
+    - --updater-email
+    help: Email used when updating RPM specification files change logs and
+      creating git commits
+    default: ''
+  updater_name:
+    option_strings:
+    - --updater-name
+    help: Name used when updating RPM specification files change logs and
+      creating git commits
+    default: ''
+  verbose:
+    option_strings:
+    - --verbose
+    - -v
+    help: Set the scripts to be verbose
+    default: False
+    action: store_true
+  work_dir:
+    option_strings:
+    - --work-dir
+    - -w
```

Comment:
Most of the options don't have short option strings, so what about making the default behavior to take the key name and add an optional `short_option_string(s)` key?:

```
work_dir:
    short_option_string: '-w' 
    help: ...
```

*I'm aware this patch-set is turning into a rabbit hole, so feel free to ignore the suggestion*